static/sass/_patterns_distro_banner.scss

```diff
@@ -53,15 +53,29 @@
     right: 2rem;
   }
 
+  .distro-banner__logo {
+    filter: brightness(40);
```

Comment:
Why do we need brightness filter for logos that are already white?

Reply:
Leftover! Will remove :)